build-system/compile/internal-version.js

```diff
@@ -19,8 +19,10 @@ const argv = require('minimist')(process.argv.slice(2));
 const {gitCommitFormattedTime} = require('../common/git');
 
 function getVersion() {
-  if (argv.rtv_version) {
-    return String(argv.rtv_version);
+  // Flag --version has been supported since 2016, so is still supported for
+  // backwards compatibility. --release_version is the documented flag.
```

Comment:
I works when used as `--version=1234567890`.

Reply:
I see. It still remains that reserved flags do not work in the common use case (`--flag value` instead of `--flag=value`), so we should avoid using them.